src/components/MenuItemList.tsx

```diff
@@ -45,16 +59,23 @@ function MenuItemList({menuItems = [], shouldUseSingleExecution = false}: MenuIt
     return (
         <>
             {menuItems.map((menuItemProps) => (
-                <MenuItem
-                    key={menuItemProps.key ?? menuItemProps.title}
-                    onSecondaryInteraction={menuItemProps.link !== undefined ? (e) => secondaryInteraction(menuItemProps.link, e) : undefined}
-                    ref={popoverAnchor}
-                    shouldBlockSelection={!!menuItemProps.link}
-                    // eslint-disable-next-line react/jsx-props-no-spreading
-                    {...menuItemProps}
-                    disabled={!!menuItemProps.disabled || isExecuting}
-                    onPress={shouldUseSingleExecution ? singleExecution(menuItemProps.onPress) : menuItemProps.onPress}
-                />
+                <OfflineWithFeedback
```

Comment:
You forgot to move the `key` prop to this component so it prints an error in console now

Reply:
sorry i don't understand. OfflineWithFeedback should be passed `key` as a prop?

Reply:
Yup, since you wrapped `MenuItem` with `OfflineWithFeedback`, you have to move the `key` prop from `MenuItem` to `OfflineWithFeedback`

<img width="425" alt="Screenshot 2024-09-18 at 22 58 37" src="https://github.com/user-attachments/assets/1c65961c-72b9-4e43-90f0-fea09e4925a0">

Reply:
nice catch, let me send a PR